src/repartidor.go

```diff
@@ -3,6 +3,7 @@ package main
 type Repartidor struct {
 	id      int
 	horario Horario
+	pedidos Pedido
```

Comment:
Aquí la variable 'pedidos' no está declarada como un array. Como no se conoce de antemano el máximo de pedidos a repartir, creo que lo mejor será hacerlo así:
```suggestion
	pedidos []Pedido
```

Reply:
Buenas @johnwaves , no se porque te ha salido la struct antigua, hice una nueva que que esa variable es pedidosPendientes que si es un array, por lo que eso ya está subsanado. Si no lo encuentras o no lo ves bien me dices. Un saludo

Reply:
Todo bien ahora, no sé por qué antes no salía. Gracias!!